snaps_openstack/provision/openstack/plugin/kolla_impl/kolla_utils.py

```diff
@@ -726,3 +729,4 @@ def _getservice_list(config):
     if config.get(consts.OPENSTACK).get(consts.SERVICES) is not None:
         service_str = config.get(consts.OPENSTACK).get(consts.SERVICES)
     return service_str
+
```

Comment:
Remove blank line at end of file.

Reply:
Blank lines at the end of a file is required for pep8 so I think it should remain.